warehouse/ingest-core/src/test/java/datawave/ingest/table/aggregator/GlobalIndexUidAggregatorExpandedTest.java

```diff
@@ -0,0 +1,719 @@
+        Value value1 = UidTestBuilder.newBuilder().withUids("uid1", "uid2", "uid3").withRemovals("uid4") // uid4 in other list
+                        .build();
+        
+        Value value2 = UidTestBuilder.newBuilder().withUids("uid4", "uid5", "uid6").withRemovals("uid7") // uid7 absent
+                        .build();
+        
+        Uid.List expectation = UidTestBuilder.valueToUidList(UidTestBuilder.newBuilder().withUids("uid1", "uid2", "uid3", "uid5", "uid6")
+                        .withRemovals("uid4", "uid7").build());
+        
+        testCombinations(expectation, asList(value1, value2));
+    }
+    
+    @Test
+    public void removeAllAcrossValue() {
+        // Do removals of all UIDs in another list work?
+        Value value1 = UidTestBuilder.newBuilder().withUids("uid1", "uid2", "uid3").withRemovals("uid4", "uid5", "uid6") // all UIDs in other list
+                        .build();
+        
+        Value value2 = UidTestBuilder.newBuilder().withUids("uid4", "uid5", "uid6").build();
+        
+        Uid.List expectation = UidTestBuilder.valueToUidList(UidTestBuilder.newBuilder().withUids("uid1", "uid2", "uid3").withRemovals("uid4", "uid5", "uid6")
+                        .build());
+        
+        testCombinations(expectation, asList(value1, value2));
+    }
+    
+    @Test
+    public void removeMultipleAbsent() {
+        // Are multiple removals persisted in partial compactions, even without matching?
+        Value value1 = UidTestBuilder.newBuilder().withUids("uid1", "uid2", "uid3").withRemovals("uid7", "uid8", "uid9") // all UIDs absent
+                        .build();
+        
+        Value value2 = UidTestBuilder.newBuilder().withUids("uid4", "uid5", "uid6").build();
+        
+        Uid.List expectation = UidTestBuilder.valueToUidList(UidTestBuilder.newBuilder().withUids("uid1", "uid2", "uid3", "uid4", "uid5", "uid6")
+                        .withRemovals("uid7", "uid8", "uid9") // all UIDs absent
+                        .build());
+        
+        testCombinations(expectation, asList(value1, value2));
+    }
+    
+    @Test
+    public void removeAllAcrossBothValues() {
+        // Do multiple removals wipe UIDs across Values and persist on partial compactions?
+        Value value1 = UidTestBuilder.newBuilder().withUids("uid1", "uid2", "uid3").withRemovals("uid4", "uid5", "uid6") // all UIDs in other list
+                        .build();
+        
+        Value value2 = UidTestBuilder.newBuilder().withUids("uid4", "uid5", "uid6").withRemovals("uid1", "uid2", "uid3") // all UIDs in other list
+                        .build();
+        
+        Uid.List expectation = UidTestBuilder.valueToUidList(UidTestBuilder.newBuilder().withUids()
+                        .withRemovals("uid1", "uid2", "uid3", "uid4", "uid5", "uid6").build());
+        
+        testCombinations(expectation, asList(value1, value2));
+    }
+    
+    @Test
+    public void removeAllAcrossValuePlusAbsent() {
```

Comment:
Nit. This test isn't removing all since UIDs 1,2,3 are never removed. Was the intent to test removeAllAcrossBothValues plus additional missing UIDs? If so, then maybe it should be something like the above test but with additional uids 7,8,9 in one or both of the withRemovals lists.

Reply:
The name was misleading so I updated it.  Thanks for the feedback.